example/lib/main.dart

```diff
@@ -51,24 +51,47 @@ class _MyAppState extends State<MyApp> {
     });
   }
 
-    void onWriteRecordButtonPress() async {
-      var creds = ClientCredentials(apiKey: "", 
-                                    apiSecret: "",
-                                    clientId: "",
-                                    publicKey: "",
-                                    privateKey: "",
-                                    publicSignKey: "",
-                                    privateSigningKey: "",
-                                    host: "",
-                                    email: "",
-                                    clientName: "");
-      var client = PluginTozny(creds);
-      try {
-        Record writtenRecord = await writeRecord(client);
-        developer.log("example flow succeeded");
-      } catch (e) {
-        developer.log("example flow failed becaused $e");
-      }
```

Comment:
There was a missing brace here causing variable declaration errors that I inadvertently fixed in writing `onWriteThenShareRecordButtonPress`. 

I did not notice this until I started a new branch off of `trunk` to start working on read record. Merging this will fix this, but at the moment `trunk` is non-functional because of it. At a glance, error was introduced in commit `7c7cd2229845d256c8a6d5f5144ce65e1ca7eb55`

Reply:
my bad ✋🏽